qfieldsync/core/preferences.py

```diff
@@ -11,4 +11,6 @@ def __init__(self):
         self.add_setting(String('exportDirectoryProject', Scope.Project, None))
         self.add_setting(String('importDirectory', Scope.Global, os.path.expanduser("~/QField/import")))
         self.add_setting(String('importDirectoryProject', Scope.Project, None))
-
+        self.add_setting(String('qfieldcloud_username', Scope.Global, None))
+        self.add_setting(String('qfieldcloud_password', Scope.Global, None))
```

Comment:
Maybe better store the token in here?
I'd rather leak that than the password I also use for [put a service you don't want anyone to know you use it].com.